packages/client/src/components/form/DocumentUploadField/DocumentUploaderWithOption.tsx

```diff
@@ -221,11 +221,17 @@ export const DocumentUploaderWithOption = (props: IFullProps) => {
       (option: ISelectOption) => option.value === documentType
     )
 
+    const options = { ...defaultOptions }
+    const resizedImage =
+      Boolean(options.maxSizeMB) &&
+      bytesToMB(uploadedImage.size) > options.maxSizeMB &&
+      (await imageCompression(uploadedImage, options))
+
```

Comment:
Thank you for your contribution! @nifaliana 🙏🏼  
I think your approach is correct, I wonder if there is any alternative other than compressing image second time to obtain the file size. One way could be utilizing the output of the `processImage` function with a small refactor. What do you think?

Reply:
Hi @tahmidrahman-dsi , 
Thanks for the suggestion :bulb:  . 
I’ve made some improvements and refactored the code as requested.
Let me know if there’s anything else!